main_train_psnr.py

```diff
@@ -164,7 +204,7 @@ def main(json_path='options/train_msrresnet_psnr.json'):
     # ----------------------------------------
     '''
 
-    for epoch in range(1000000):  # keep running
+    for epoch in range(500000):  # keep running
```

Comment:
this is now how you change the max iteration. it's in the config

Reply:
There is no max iteration stored in the config file. I have prepared a modified config if we want this.

Reply:
Good point!
As discussed add it to the config file `train_swinir_sr_lightweight.json`